src/BibTeXCitations.ts

```diff
@@ -34,15 +34,31 @@ export class BibTeXCitations {
       if (pos == null) {
         return Promise.reject("Cannot get location");
       } else {
-        const uri = vscode.Uri.file(this.cfg().bibTeXFilePath);
+        const uri = vscode.Uri.file(this.bibTexFilePath());
         return new vscode.Location(uri, pos);
       }
     });
   }
 
-  private static bibTeXFile(): Promise<string> {
+  private static bibTexFilePath(): string {
     const path = this.cfg().bibTeXFilePath;
-    if (path == null) {
+
+    // Absolute path (Unix and Windows)
+    if (path.startsWith("/") || path.indexOf(":\\") == 1) {
+      return path;
+    }
+
+    // Workspace relative path
+    const folders = vscode.workspace?.workspaceFolders;
+    if (folders && folders.length > 0) {
+      return folders[0].uri.fsPath.toString() + "/" + path;
```

Comment:
This is doing a path join, right? like using the folder path + the filename?

If so, I'll swap out with and use `import { join } from 'path';` so it also works on Windows, once I merge

Reply:
Yes that is the intention, i'm clearly not used to programming for windows 😓